src/Illuminate/Auth/AuthManager.php

```diff
@@ -80,23 +80,19 @@ public function guard($name = null)
      */
     protected function resolve($name)
     {
-        $config = $this->getConfig($name);
-
-        if (is_null($config)) {
+        if (is_null($config = $this->getConfig($name))) {
```

Comment:
I think there are many other inline assignments in the project.
Or do you mean here ?